### PR TITLE
feat: wip

### PR DESCRIPTION
**Pull request by elijahmg**

### CODE REVIEW

File: src/components/wrappers/admin-quiz-wrapper.tsx
Comment (martinmachma) [inline src/components/wrappers/admin-quiz-wrapper.tsx:1]: Is this change intended?

File: src/pages/[quizId]/overview/index.tsx [hunk 1]
```diff
@@ -7,13 +7,20 @@ import OverviewQuestion from '../../../components/overview-question';
...
-  const { teamData, roundId } = useTeamQuizDataStore((state) => ({
+  const router = useRouter();
+
+  const { teamData, roundId, quizData } = useTeamQuizDataStore((state) => ({
     teamData: state.teamData,
     roundId: state.quizData.quizStatus?.currentQuestion.roundId,
+    quizData: state.quizData,
   }));
 
+  console.log({ quizData });
```
Comment (martinmachma) [inline src/pages/[quizId]/overview/index.tsx]: Remove

File: src/pages/admin/[quizId]/round-overview.tsx [hunk 2]
```diff
@@ -15,23 +15,64 @@ import {
...
+    trpc.admin.updateCurrentQuestion.useMutation({ onError: handleTRPCError });
+
+  const { mutate: updateQuizStatus } = trpc.admin.updateQuizStatus.useMutation({
+    onSuccess: () => handOnSuccessfullyUpdatedQuizStatus(),
+    onError: handleTRPCError,
+  });
+
+  const isNextRoundExist = !!quizState.rounds?.[roundIndex + 1];
+
+  const handOnSuccessfullyUpdatedQuizStatus = () => {
```
Comment (martinmachma) [inline src/pages/admin/[quizId]/round-overview.tsx]: You probably wanted to name this handleOnSuccessfullyUpdatedQuizStatus?

**Review by martinmachma — APPROVED** [review]

File: src/components/wrappers/admin-quiz-wrapper.tsx
Comment (elijahmg) [reply src/components/wrappers/admin-quiz-wrapper.tsx:1]: Kinda :)
